templates/cronjob.yaml

```diff
@@ -0,0 +1,202 @@
+{{- if .Values.garbageCollect.enabled }}
+apiVersion: batch/v1
+kind: CronJob
+metadata:
+  name: {{ template "docker-registry.fullname" . }}-garbage-collector
+  labels:
+    app: {{ template "docker-registry.name" . }}
+    chart: {{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}
```

Comment:
:bulb: **Note:** Looks like `metadata.labels.chart` isn't being set consistently across all resources in the chart. The labels lack prefixes, anyway, making them problematic for some observability tooling (e.g: Elasticsearch will drop log events that include a field with a type conflicting with the index's field type mapping, and the `app` label conflicts with every one of Kubernetes's [recommended labels](https://kubernetes.io/docs/concepts/overview/working-with-objects/common-labels/)).

No changes requested, just making a note of this for reference.

Reply:
👍 for now, I'll leave label/annotation consistency out of this PR